sql/python/sqlflow_submitter/db.py

```diff
@@ -82,6 +84,8 @@ def reader():
                             cell = row[field_names.index(name)]
                     features.append(cell)
                 yield (tuple(features), [label])
+            if len(rows) < fetch_size:
```

Comment:
Good point! Should merge with https://github.com/sql-machine-learning/sqlflow/pull/609